repositories/transfercheck_partners.go

```diff
@@ -14,15 +15,20 @@ func selectPartners() squirrel.SelectBuilder {
 		From(dbmodels.TABLE_PARTNERS)
 }
 
-func (repo MarbleDbRepository) ListPartners(ctx context.Context, exec Executor) ([]models.Partner, error) {
+func (repo MarbleDbRepository) ListPartners(ctx context.Context, exec Executor, filters models.PartnerFilters) ([]models.Partner, error) {
 	if err := validateMarbleDbExecutor(exec); err != nil {
 		return nil, err
 	}
 
+	query := selectPartners().OrderBy("created_at DESC")
+	if filters.Bic.Valid {
+		query = query.Where(squirrel.Eq{"LOWER(bic)": strings.ToLower(filters.Bic.String)})
+	}
```

Comment:
Here bic is "lowered" but on create + update it is not.
I know the index is created using "lower", my question: does this "where" use the index or the stored "non lowered" value in the column ?

Reply:
Good question :-) 
The answer you're looking for is that since I created the index on a computed field (not on the raw column value), it should use the index to apply the filter.
The slightly longer answer is that because the partners table is small, and is expected to remain kinda small, it's possible that the DB still decides "fuck the index I can scan 10 rows" (especially if you SELECT fields that are not in the index). 
I still created the index for the sake of completeness.

Reply:
that being said, you're right it's a bit dumb that I UPPER the bic on insertion and LOWER it on filtering (though technically it works just the same). I'll change that.